### ISSUE TITLE
questions on accuracy/edit_distance 

### ISSUE THREAD

**Issue by MengXinChengXuYuan** [breadbread1984/R3DCNN#15]
Hi!
Thank you so much for your awesome work first!
I get some questions on the training process however:

1.  how long does it take to get a 'good' model? I mean, it seems the model converge very slowly? I just pre-trained  the model provided as indicated in the readme, loss decrease very fast in the first n steps and will hang around 2-3 for quiet a while, is that normal? Or I just missed some thing?

2.  I tried the model which I can get the best for now, while the steps are not so much, with loss around 3, the mean_edit_distance is 1. Also I tried to calculate the accuracy and I it rest 0... （I can't understand how can the acc be 0... Even if the weights are random, the output will always get some correct hits by accident, right?）I used the code from @ #6 , sorry I'm quite new to tf (mainly using pt), I don't know if it's the model always outputs 'none' case it didn't converge or there's something wrong in the scripts?

Hoping for your response! @breadbread1984 @breadbread1984

**Comment by MengXinChengXuYuan** [comment]
Hi I know this maybe rude but I still have to ask ...
Are you sure that using the pre-trained model and your code, after training for 20000 steps, it will get a model that converges on the test set?

Sorry but I just followed all the instructions on readme and used the pre-trained C3D provided by another repo, after 20000 steps the model converges good on the train set but stays at loss of 2 on the test set

I don't know where could be wrong, or what could be missed...  And I found there are many people facing the same problem as I do, so I really wonder if you can actually get a model achieving 83% accuracy on the test set. 

Anyway, I'm quite thankful to your great work! It helps me better understand the paper

One thing in your work which is certainly wrong is that in the paper, for online prediction of untrimmed streams, CTC is only involved as a loss for the LSTM in the training phase. In the test phase the author do not use CTC beam search for the final prediction, but simply use the output of LSTM of every clip (8 frames)

**Comment by breadbread1984** [comment]
this is a casual time project. I can't guarantee its performance. thx for your correction for the implement. any pull request is appreciated.

**Comment by buaa-luzhi** [comment]
@MengXinChengXuYuan
Hello, are you still working on this problem. Can you get better results?
Thanks
